src/domains/sign/io_sign_support_func.tf

```
@@ -0,0 +1,183 @@
+locals {
+  io_sign_support_func = {
+    app_settings = {
+      WEBSITE_VNET_ROUTE_ALL                          = "1"
+      WEBSITE_ADD_SITENAME_BINDINGS_IN_APPHOST_CONFIG = "1"
+      WEBSITE_DNS_SERVER                              = "168.63.129.16"
+      WEBSITE_RUN_FROM_PACKAGE                        = "1"
```

```suggestion
      WEBSITE_ADD_SITENAME_BINDINGS_IN_APPHOST_CONFIG = "1"
      WEBSITE_DNS_SERVER                              = "168.63.129.16"
      WEBSITE_RUN_FROM_PACKAGE                        = "1"
```
already set in the function module https://github.com/pagopa/terraform-azurerm-v3/blob/main/function_app/main.tf#L298